modules/ROOT/pages/modify-third-party-settings.adoc

```diff
@@ -1,8 +1,8 @@
-= Modifying Third Party Connection Settings
+= Modifying Third-Party Connection Settings
 
-You can modify third party connection settings including contact details, identifiers, digital certificates, and more.
+You can modify third-party connection settings including contact details, identifiers, digital certificates, and more.
```

Comment:
This is my pickyism: "and more" adds nothing, because "including" implies that the list is not complete, and "more" doesn't tell me anything.